__tests__/app.test.js

```diff
@@ -73,6 +73,61 @@ describe("App", () => {
           });
         });
     });
+    // Query tests
+    test("200: Responds with the array of review objects, all of which only have the category of 'dexterity'", () => {
+      return request(app)
+        .get("/api/reviews?category=dexterity")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          reviews.forEach((review) => {
+            expect(review).toHaveProperty("category", "dexterity");
+          });
+        });
+    });
+    test("200: Responds with the array of review objects sorted by comment_count (using default order)", () => {
+      return request(app)
+        .get("/api/reviews?sort_by=comment_count")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(reviews).toBeSortedBy("comment_count", { descending: true });
+        });
+    });
+    test("200: Responds with the array of review objects ordered by ascending (using default sort_by)", () => {
+      return request(app)
+        .get("/api/reviews?order=asc")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(reviews).toBeSortedBy("created_at", { descending: false });
+        });
+    });
+    test("404: Category not found", () => {
+      return request(app)
+        .get("/api/reviews?category=test")
+        .expect(404)
+        .then(({ body: { message } }) => {
+          expect(message).toBe("Category 'test' Not Found");
+        });
+    });
+    test("400: Invalid sort_by query", () => {
+      return request(app)
+        .get("/api/reviews?sort_by=test")
+        .expect(400)
+        .then(({ body: { message } }) => {
+          expect(message).toBe(
+            "Bad Request: Enter a valid sort_by query (See endpoints.md)"
+          );
+        });
+    });
+    test("400: Invalid order query", () => {
+      return request(app)
+        .get("/api/reviews?order=test")
+        .expect(400)
+        .then(({ body: { message } }) => {
+          expect(message).toBe(
+            "Bad Request: Enter a valid order query (asc|desc)"
+          );
+        });
+    });
   });
 
```

Comment:
What about a 200 for a valid category query but no reviews so it should have an empty array of reviews?

Reply:
e.g. 'childrens games'?